src/components/Modeler.vue

```diff
@@ -132,6 +132,7 @@ import registerInspectorExtension from '@/components/InspectorExtensionManager';
 import initAnchor from '@/mixins/linkManager.js';
 
 const version = '1.0';
+const unsupportedElements = ['documentation', 'extensionElements'];
```

Comment:
I'm not sure about moving this constant to here. In general I think variables should be declared as close to where they're used as possible. I think this array is only used in the single method below, right?